docs/style-guide.md

```diff
@@ -0,0 +1,47 @@
+# Style Guide
+
+When contributing to the documentation, please ensure you follow [the AsciiDoc recommended practices].
+They are not currently rigidly enforced.
+However, this may change in the future.
+
+## Key Terms, Jargon, and Terminology
+
+- AMD64
+- ARM32
+- ARMv7
+- ARMv8
+- AWS
+- Apache
+- AppArmor
+- Bash
+- Bitbucket
+- Cron
+- Docker
+- HMAC
+- HTTP
+- HTTPS
+- HashiCorp
+- JSON
+- Kubernetes
+- Linux
+- MySQL
+- NGINX
+- OAuth
+- PostgreSQL
+- Redis
+- AWS S3
+- SELinux
+- SQLite
+- SSL
+- STDERR
+- STDIN
+- STDOUT
+- Traefik
+- localhost
+- webhooks
+- OpenSSL
+- RPC
+- URL
+- GDPR
+
+[the AsciiDoc recommended practices]: https://asciidoctor.org/docs/asciidoc-recommended-practices/
```

Comment:
You may also want to link our best practices guide,
because you have to imlement bp with attributes and directives :smile:

Reply:
What's "bp"?

Reply:
On reviewing the best practices guide, I don't see a natural way to link them. I'm open to suggestions though.

Reply:
as an external link: https://github.com/owncloud/docs/blob/master/docs/best-practices.md

Reply:
Thanks for the suggestion. I've updated the previous commit to include it.